learning_journal/views.py

```diff
@@ -1,35 +1,60 @@
-# from pyramid.response import Response
-# from pyramid.view import view_config
-
-# from sqlalchemy.exc import DBAPIError
-
-# from .models import (
-#     DBSession,
-#     Entry,
-#     )
-
-
-# @view_config(route_name='home', renderer='templates/mytemplate.pt')
-# def my_view(request):
-#     try:
-#         one = DBSession.query(Entry).filter(Entry.title == 'one').first()
-#     except DBAPIError:
-#         return Response(conn_err_msg, content_type='text/plain', status_int=500)
-#     return {'one': one, 'project': 'learning-journal'}
-
-
-# conn_err_msg = """\
-# Pyramid is having a problem using your SQL database.  The problem
-# might be caused by one of the following things:
-
-# 1.  You may need to run the "initialize_learning-journal_db" script
-#     to initialize your database tables.  Check your virtual
-#     environment's "bin" directory for this script and try to run it.
-
-# 2.  Your database server may not be running.  Check that the
-#     database server referred to by the "sqlalchemy.url" setting in
-#     your "development.ini" file is running.
-
-# After you fix the problem, please restart the Pyramid application to
-# try it again.
-# """
+# -*- coding: utf-8 -*-
+from pyramid.view import view_config
+from .models import Entry, DBSession
+from wtforms import Form, StringField, TextAreaField, validators
+from pyramid.httpexceptions import HTTPFound
+from jinja2 import Markup
+import markdown
+
+
+@view_config(route_name='index_route', renderer='templates/list.jinja2')
+def post_index(request):
+    entries = DBSession.query(Entry).all()
+    return {'entries': entries}
```

Comment:
good.  Might be better to hand back the query itself, rather than the list of entries you get from `.all()`, but that's not a big difference.